<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Storage</class>
 <widget class="QWidget" name="Storage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="StudentScoreList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>530</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="StudentNumberEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ChineseEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>380</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Modify">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MathEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EnglishEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>240</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>语文</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数学</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>英语</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>460</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Close</sender>
   <signal>clicked()</signal>
   <receiver>Storage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
